content/neuvector-prime/azure/contents.lr

```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+To Update the NeuVector version to the latest version supported by the marketplace listing, via the "az k8s-extension update" command.
+
+Execute the following commands in a Cluster Cloud Shell:
 
 -  ---
    **NOTE** if cluster is deployed in westcentralus location, execute the following command.
            This command will also work on other regions once the feature is rolled out to other regions.
```

Comment:
not part of this change, but "is rolled out to other regions" should be "is rolled out to those regions"